src/components/two-factor-authentication/SelfServiceWorkflow.vue

```diff
@@ -0,0 +1,255 @@
+<script setup lang="ts">
+  import { computed, ref, type ComputedRef, type Ref } from 'vue';
+  import { useDisplay } from 'vuetify';
+  import LayoutCard from '@/components/cards/LayoutCard.vue';
+  import SpshTooltip from '@/components/admin/SpshTooltip.vue';
+  import { useI18n, type Composer } from 'vue-i18n';
+  import {
+    useTwoFactorAuthentificationStore,
+    type TwoFactorAuthentificationStore,
+  } from '@/stores/TwoFactorAuthentificationStore';
+
+  const { mdAndDown }: { mdAndDown: Ref<boolean> } = useDisplay();
+  const { t }: Composer = useI18n({ useScope: 'global' });
+  const twoFactorStore: TwoFactorAuthentificationStore = useTwoFactorAuthentificationStore();
+
+  const workflowStep: Ref<'start' | 'qrcode' | 'verify'> = ref('start');
+  const errorMessage: Ref<string> = ref('');
+  const otp: Ref<string> = ref('');
+
+  const dialogHeader: ComputedRef<string> = computed(() => {
+    switch (workflowStep.value) {
+      case 'start':
+        return t('admin.person.twoFactorAuthentication.setUpLong');
+      case 'qrcode':
+        return t('admin.person.twoFactorAuthentication.softwareTokenOption');
+      case 'verify':
+        return t('admin.person.twoFactorAuthentication.header');
+      default:
+        return t('admin.person.twoFactorAuthentication.setUpLong');
+    }
+  });
+
+  type Emits = {
+    (event: 'dialogClosed'): void;
+  };
+
+  const emits: Emits = defineEmits<{
+    (event: 'dialogClosed'): void;
+  }>();
+
+  type Props = {
+    personId: string;
+  };
+
+  const props: Props = defineProps<Props>();
+
+  async function close2FADialog(isActive: Ref<boolean>): Promise<void> {
+    if (workflowStep.value !== 'start') {
+      emits('dialogClosed');
+      twoFactorStore.qrCode = '';
+      twoFactorStore.hasToken = null;
+      twoFactorStore.tokenKind = null;
+    }
```

Comment:
Wenn Ich in der "Verify" step bin und nichts eingebe und stattdessen dann den Dialog schliesse, berücksichtigt mich das System 2FA als eingestellt... Wieso?

Reply:
Laut Ticket soll 2FA nur als eingestellt gelten, wenn ein OTP eingegeben wird. 
![image](https://github.com/user-attachments/assets/34967033-746c-4da9-bf63-3f909962c337)

Kann das ein BE Problem sein?

Reply:
Dafür gab es eine Anmerkungen oben in den Notes :D ich muss da einmal was bei PrivacyIDEA umstellen. Da die Umstellung, aber den aktuellen Stand von main kaputt machen würde, würde ich das nur kurzfristig für die Tests umstellen. Also sag einfach kurz Bescheid, wenn du das testen willst

Reply:
Danke